internal/services/executor/executor.go

```diff
@@ -883,6 +883,8 @@ func (e *Executor) setupTask(ctx context.Context, rt *runningTask) error {
 	}
 
 	_, _ = outf.WriteString("Starting pod.\n")
+	ctx, cancel := context.WithTimeout(ctx, time.Duration(time.Minute*30))
```

Comment:
Can you make this a const and also add some comments describing why it was set to 30 minutes?

Reply:
Hi @sgotti what value of timeout do you think is better? Thanks